src/constants/endpoints.ts

```diff
@@ -10,6 +10,7 @@ export interface Discipline {
   parentTitle: string | null // if null then it is the root discipline
   apiKey?: string
   needsTrainingSetEndpoint: boolean
+  leafDisciplines?: number[]
```

Comment:
Hmm these different api responses are really ugly. I think all disciplines, no matter whether they are modules, professions, individual areas or whatever else should have the same responses. Same for all documents.